build/project_cleanup.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 3600s
+
+options:
+  machineType: "N1_HIGHCPU_8"
+
+steps:
+- name: 'gcr.io/cloud-builders/gcloud'
+  entrypoint: 'bash'
+  args: ["scripts/project_cleanup.sh"]
+  id: Project deletion
+  env:
+  - "FOLDER_ID=${_FOLDER_ID}"
+options:
+  logging: CLOUD_LOGGING_ONLY
```

Comment:
Why setting this? What's the behavior if omitting this?

Reply:
Since I am using a user-specified service account for this, I cannot use the default logging option. The three options I can choose are CLOUD_LOGGING_ONLY, GCS_ONLY (specified user-created cloud bucket) or NONE.
https://cloud.google.com/build/docs/securing-builds/store-manage-build-logs

Reply:
gotcha, yea, it's required if using custom SA. Can you add a comment above it to mention that?

Reply:
Done